<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拓扑结构</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计  算</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>240</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>谐波计算</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>111111111111</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>111111111111</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>111111111111</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_6"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确   定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示计算结果</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置计算参数</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>供电臂长度L/m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>分段长度l/Km</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>链式网络模型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>环境（大地导电率ρ）</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>240</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>短路阻抗计算</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>短路点距离S/Km</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>短路导线1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>短路导线2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确   定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_1</zorder>
  <zorder>lineEdit_1</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBox_1</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
